src/transformSvg.js

```diff
@@ -43,8 +43,10 @@ export default t => ({
       // <svg stroke-width="5">
       // to
       // <svg strokeWidth="5">
-      // don't convert any custom data-* attributes
-      if (!path.node.name.name.startsWith('data-')) {
+      // don't convert any custom data-* or aria-* attributes just wrap in quotes
+      if (/^(data-|aria-)/.test(path.node.name.name)) {
```

Comment:
nit: it would be nice to make this group non-capturing, e.g. `/^(?:data-|aria-)/`

Reply:
`/^data-|aria-/` will only try and match `data-` at the start of the string, so would return true if `aria-` existed anywhere in the string (should be pretty unlikely to be fair!)

Could do `/^data-|^aria-/` to avoid the use of groups if that's preferred?

Reply:
@emortlock oops, yes, that's what i meant :-) yeah, let's go with that